service-commands/scripts/.env/.aliases/10.docker_aliases

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+alias d='docker'
+alias dps='d ps'
+alias dl='d logs'
+function de () {
+    d exec -ti ${@} bash
+}
+function des () {
+    d exec -ti ${@} sh
+}
+
+alias dc='docker-compose'
+alias dcps='dc ps'
+alias dcr='dc run'
+alias dcs='dc run --entrypoint bash'
+alias dcsh='dc run --entrypoint sh'
+alias dcu='dc up'
+alias dcl='dc logs -f --tail 5'
+alias dcud='dc up -d'
+alias dcb='dc build'
+alias dcd='dc down'
+
+alias docker-prune="docker rm \$(docker ps --all -q -f status=dead); docker system prune; docker network prune"
+alias docker-port-nuke="sudo systemctl stop docker; sudo rm -rf /var/lib/docker/containers/*; sudo systemctl start docker"
+
+function docker-rm-container() {
```

Comment:
nit but we have underscored fn names in some files and hyphens here - would be good to consolidate!

Reply:
Thanks! It turns out some shells don't like hyphens, so they're all underscores now!

https://unix.stackexchange.com/questions/168221/